### ISSUE TITLE
"Critical Injection Failure"

### ISSUE THREAD

**Issue by Derpford** [Jackiecrazy/footwork#3]
I'm not sure if this is Improved Damage Framework being incompatible with this, but here's a crashlog:
https://gist.github.com/Derpford/4bb9332524ebaf14f6edc8fbd69a5572

**Comment by Jackiecrazy** [comment]
Indeed. Improved Damage Framework has an excessive number of redirects, overwrites, and base level changes, which makes it likely to be incompatible with a large number of mods that it doesn't give explicit compat for. I don't think there's much I can do about that.
